src/components/Bounty/BountyCreateForm.tsx

```diff
@@ -206,26 +209,50 @@ export function BountyCreateForm() {
   const currency = form.watch('currency');
   const unitAmount = form.watch('unitAmount');
   const nsfwPoi = form.watch(['nsfw', 'poi']);
-  const [creating, setCreating] = useState(false);
   const requireBaseModelSelection = [
     BountyType.ModelCreation,
     BountyType.LoraCreation,
     BountyType.EmbedCreation,
   ].some((t) => t === type);
 
+  const { conditionalPerformTransaction } = useBuzzTransaction({
+    message: (requiredBalance) =>
+      `You don't have enough funds to create this bounty. Buy ${numberWithCommas(
+        requiredBalance
+      )} more BUZZ to perform this action.`,
+    performTransactionOnPurchase: false,
```

Comment:
This was intentional, just want the user to double-confirm they wanna create the bounty I guess.